lib/openstax/aws/stack.rb

```diff
@@ -338,6 +338,52 @@ def status
       end
     end
 
+    def status_reason
+      begin
+        aws_stack.stack_status_reason
+      rescue Aws::CloudFormation::Errors::ValidationError => ee
+        case ee.message
+        when /Stack.*does not exist/
+          "DOES_NOT_EXIST"
+        else
+          raise
+        end
+      end
+    end
+
+    def events
+      begin 
+        aws_stack.events()
+      rescue
+        raise "something's wrong sorry"
+      end
+    end
+
+    def latest_failed_events
+      latest_deployment_reasons = []
+      begin
+        ###search in latest deployment events
+        aws_stack.events.each do |event|
+          if(event.data.resource_status.include?("_FAILED") || (event.data.resource_status === "ROLLBACK_IN_PROGRESS" && event.data.resource_status_reason))
```

Comment:
Is there a reason why `===` instead of `==`? (same few lines below)
Also, i think it would be nice to format it a little since github adds a scroll for lines over 120 characters long